src/amchart/CommonSerial.js

```diff
@@ -126,8 +82,77 @@
         return retVal;
     };
 
+    // Axes
+    //CommonSerial.prototype._origXAxis = CommonSerial.prototype.xAxis;
+    //TODO need to figure out how to allow overwriting of 
+    // also if we dont do the override then we can shorted the _orgiXAxis call and rename that function
+    CommonSerial.prototype.xAxis = function (_) {
+        if (!arguments.length) {
+            var context = this;
+            var axes = CommonSerial.prototype._origXAxis.call(this);
+            axes.forEach(function(axis) {
+                axis._context = context;
+            });
+            return axes;
+            //return  // might need to override _context b4 passing ...  but so far dont need to
+        }
+        if (CommonSerial.prototype._origXAxis.call(this)[_]) {
+             var axis = CommonSerial.prototype._origXAxis.call(this)[_]
+             axis._context = this;
+             return axis; // might need to override _context b4 passing ...  but so far dont need to
+        } else {
+            console.log('bob')
+            console.log(CommonSerial.prototype._origXAxis())
+            console.log('/bob')
+            var axis = new Axis();
+            axis._context = this;
+            var currentAxes = CommonSerial.prototype._origXAxis.call(this);
+            currentAxes.push(axis);
+            CommonSerial.prototype._origXAxis.call(this, currentAxes)
+        } 
+        return axis;
+    };
+
+    //CommonSerial.prototype._origYAxis = CommonSerial.prototype.yAxis;
```

Comment:
a cleaner version of above (depending on a few things i will remove the prototype calls)